apps/sharedtest/test/unit/event_safety_test.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+/* global eventSafety */
+
+require('/shared/js/event_safety.js');
+
+suite('eventSafety', function() {
+
+  test('fires after event', function(done) {
+    this.sinon.useFakeTimers();
+    eventSafety(window, 'foobar', done, 1000);
```

Comment:
The problem here is that asserting on ```true``` will never trigger the message. It makes it show up in the success case, but I'm not sure how valuable that is. I'll go ahead and add it regardless though to get this landed.